pkg/account/main.go

```diff
@@ -4,15 +4,15 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
```

Comment:
You left quite a few things in `pkg`. I'd rather remove them as well and copy them over when needed.